x/oracle/keeper/bounty.go

```diff
@@ -0,0 +1,57 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/shentufoundation/shentu/v2/x/oracle/types"
+)
+
+// SetCreatorLeftBounty This function saves the LeftBounty struct associated to its address in the store.
+func (k Keeper) SetCreatorLeftBounty(ctx sdk.Context, leftBounty types.LeftBounty) {
```

Comment:
just `SetRemainingBounty` should be better. same with all the other ones.

Reply:
ok